core/util.go

```diff
@@ -26,9 +26,11 @@ import (
 	"io"
 	"io/ioutil"
 	"math/big"
+	mrand "math/rand"
```

Comment:
Because we also import crypto/rand

On Thu, Oct 22, 2015 at 12:02 PM bifurcation <notifications@github.com>
wrote:

> In core/util.go
> <https://github.com/letsencrypt/boulder/pull/1027#discussion_r42791394>:
>
> > @@ -26,9 +26,11 @@ import (
> >  	"io"
> >  	"io/ioutil"
> >  	"math/big"
> > +	mrand "math/rand"
>
> Why the custom package name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1027/files#r42791394>.
>